kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/source/S3SourceTaskTest.scala

```diff
@@ -100,7 +100,61 @@ class S3SourceTaskTest
           val props = DefaultProps
             .combine(
               Map(
-                "connect.s3.kcql" -> s"insert into ${bucketSetup.TopicName} select * from $BucketName:${bucketSetup.PrefixName}/$dir/ STOREAS `${format.entryName}$formatExtensionString` LIMIT 190",
+                KCQL_CONFIG -> s"insert into ${bucketSetup.TopicName} select * from $BucketName:${bucketSetup.PrefixName}/$dir/ STOREAS `${format.entryName}$formatExtensionString` LIMIT 190",
```

Comment:
My only thought on replacing the values (eg "connect.s3.kcql") in the test with the constant, is it doesn't protect us if the constant value changes between versions, which is why it was originally implemented without referencing the constants.  But this is minor and is hopefully covered by other tests.

Reply:
Sorry, I fail to fully understand. If we have the text there it protects from a change in KCQL_CONFIG? is that the benefit?

Reply:
that was the gist of it yes

Reply:
Wouldn't a class with one test cover that? rather than use the actual string key and not the constant?